recipes/fermikit/test-data/test.fastq

```diff
@@ -0,0 +1,100 @@
+@phiX174_29_494_0_1_0_0_2:0:0_3:0:0_0
```

Comment:
As I said, the perl disaster has different reasons. It's somewhere on my todo list :(